src/GraphQL.Tests/Execution/AstFromValueTests.cs

```diff
@@ -70,63 +70,38 @@ public void converts_int_to_int_value()
         public void converts_double_to_float_value()
         {
             double val = 0.42;
-            var result = val.AstFromValue(null, new FloatGraphType());
+            var result = val.AstFromValue(new FloatGraphType());
```

Comment:
I would inverse this method to `ScalarGraphType.AstFromValue(object value)` to narrow down the scope.

Reply:
Are you suggesting this?

```cs
static class GraphQLExtensions
{
    //old
    public static IValue AstFromValue(this object value, IGraphType type);
    //suggestion 1
    public static IValue AstFromValue(this IGraphType type, object value);
    //suggestion 2
    public static IValue ToAst(this IGraphType type, object value);
}
```

Reply:
committed suggestion 2...please review

Reply:
Yes, 2. Or make `ToAst` as instance method, not extension.